packages/scandipwa/src/style/abstract/_media.scss

```diff
@@ -75,7 +75,17 @@
 }
 
 @mixin hoverable {
-    @media (hover: hover) and (pointer: fine) { 
+    @media (hover: hover) and (pointer: fine) {
+        @content;
+    }
+}
+
+@mixin pointer-device-coarse {
+    @media (pointer: coarse) {
+        @content;
+    }
+
+    @media (pointer: none) {
```

Comment:
Please use the `OR` syntax instead of duplicating the block.
https://css-tricks.com/logic-in-css-media-queries/#aa-or